packages/server/test/integration/cypress_spec.coffee

```diff
@@ -828,6 +828,7 @@ describe "lib/cypress", ->
           groupId: null
           specs: specFiles
           specPattern: specPattern
+          browser: browser
         }
 
         @createRun = @sandbox.stub(api, "createRun").withArgs(createRunArgs)
```

Comment:
later I will add https://github.com/bahmutov/not-allowed to catch other `withArgs` invocations that caused me so much grief